pip_review/__main__.py

```diff
@@ -244,55 +136,59 @@ def confirm(question):
     return answer == 'y'
 
 
+def get_outdated_packages(local=False, pre_release=False, user=False):
+    command = ['pip', 'list', '--outdated', '--disable-pip-version-check']
+    if local:
+        command.append('--local')
+    if pre_release:
+        command.append('--pre')
+    if user:
+        command.append('--user')
+    if parse_version(pip.__version__) > parse_version('9.0'):
+        command.append('--format=json')
+        output = check_output(" ".join(command)).decode('utf-8')
+        packages = json.loads(output)
+        return packages
+    else:
+        output = check_output(" ".join(command)).decode('utf-8').strip()
+        packages = []
+        for line in output.splitlines():
+            package = {}
+            line = line.split(" - ")
+            package['name'] = re.findall(r'^[a-zA-Z0-9\-]+', line[0])[0]
+            package['version'] = re.findall(r'\(([0-9a-zA-Z\.]+)\)', line[0])[0]
+            package['latest_version'] = re.findall(r'(^[0-9a-zA-Z\.]+)', line[1].split(":")[1].strip())[0]
+            packages.append(package)
```

Comment:
Would be nice to factor out the above lines (154-161) into a new function and call it `parse_legacy` or something.